### PR TITLE
[Issue #95]: change newsletter to subscribe terminology

### LINKED ISSUE #95: [Task]: Change "Newsletter" to "Subscribe"
### Summary

https://github.com/HHS/simpler-grants-gov/issues/1560

The page at https://simpler.grants.gov/newsletter should be updated to use the term "subscribe" instead of "newsletter" (as the later implies regular cadence). 

- Change the page route to `https://simpler.grants.gov/subscribe`
- Redirect `/newsletter` to `/subscribe` to preserve old links floating around (Note: Maybe this shouldn't be a 301/308 permanent redirect, but just handled in the app, as it's possible we'll want the `/newsletter` some time in the future — eng to sort out best option)
- Change the site navigation item to "Subscribe" 
- Change the breadcrumb to "Subscribe" 
- Change the page title (`h1`) to "Subscribe to project updates" 
- Change the intro paragraph to "If you sign up for the Simpler.Grants.gov product updates, we’ll keep you informed…"
- Change the disclaimer to "The Simpler.Grants.gov email subscriptions are powered by the Sendy…"

### Open questions: 
- Is "Updates" or "Subscribe" or "Signup" a better CTA for the URL / menu? The `/process` and `/research` pages have a "Sign up for project updates" CTA.

### Acceptance criteria

- [ ] The term "newsletter" is no longer used in production 
- [ ] Be sure to check that all the content is updated ("find all" in code for `newsletter`) 
- [ ] match the mockup [from comment below](https://github.com/HHS/simpler-grants-gov/issues/1560#issuecomment-2023851171)

### PR DESCRIPTION
**Pull request by rylew1**
## Summary
Fixes #95 

## Changes proposed

- Most of the newsletter references become `subscribe`
- `/newsletter/confirmation` => `/subscribe/confirmation`
- `/newsletter/unsubscribe` => `/unsubscribe` (top level route)
- Updated tests and translation strings

https://github.com/navapbc/simpler-grants-gov/assets/93001277/00417107-95c3-4be0-a4d4-07b45cd0d667

### CODE REVIEW

File: frontend/src/i18n/messages/en/index.ts [hunk 1]
```diff
@@ -280,15 +280,15 @@ export const messages = {
...
     },
   },
-  Newsletter: {
+  Subscribe: {
     page_title: "Newsletter | Simpler.Grants.gov",
-    title: "Newsletter signup",
+    title: "Subscribe to project updates",
     intro: "Subscribe to get Simpler.Grants.gov project updates in your inbox!",
     paragraph_1:
       "If you sign up for the Simpler.Grants.gov newsletter, we’ll keep you informed of our progress and you’ll know about every opportunity to get involved.",
```
Comment (sumiat) [inline frontend/src/i18n/messages/en/index.ts]: Could you change this to "If you sign up for the Simpler.Grants.gov project updates, we'll keep you..." ? Replacing the word "newsletter."

**Review by sumiat — COMMENTED** [review]
Just made one comment to update the word "newsletter" on the subscribe page. Otherwise, LGTM!

File: frontend/src/i18n/messages/en/index.ts (hunk 1 above)
Comment (rylew1) [reply frontend/src/i18n/messages/en/index.ts]: good catch - here's the update:
image

Comment (rylew1) [reply frontend/src/i18n/messages/en/index.ts]: Note that the title of pages still shows Newsletter 
image

Comment (sumiat) [reply frontend/src/i18n/messages/en/index.ts]: Oh, good catch! we'll want to change that! I realize this is different than `subscribe` and @andycochran and @crystabelrangel may have different thoughts on this, but could we change that to say "Mailing List"? We can always change it something else if Design thinks it's better as a different name.

Comment (crystabelrangel) [reply frontend/src/i18n/messages/en/index.ts]: "Project Updates"? Not sure if that is too redundant.

**Comment by acouch** [comment]
Closing for now, we might pick this up later.

**Pull request closed without merging**